ghost/i18n/locales/ar/portal.json

```diff
@@ -1,209 +1,210 @@
+    "Too many attempts try again in {{number}} days.": ".أيام{{number}} محاولات متعددة جدًا، حاول مرة أخرى بعد ",
+    "Too many attempts try again in {{number}} hours.": ".ساعات {{number}} محاولات متعددة جدًا، حاول مرة أخرى بعد",
+    "Too many attempts try again in {{number}} minutes.": ".دقيقة {{number}} محاولات متعددة جدًا، حاول مرة أخرى بعد ",
+    "Too many different sign-in attempts, try again in {{number}} days": ".أيام{{number}} محاولات تسجيل الدخول متعددة جدًا، حاول مرة أخري بعد ",
+    "Too many different sign-in attempts, try again in {{number}} hours": ".ساعة {{number}} محاولات تسجيل الدخول متعددة جدًا، حاول مرة أخري بعد ",
+    "Too many different sign-in attempts, try again in {{number}} minutes": ".دقيقة {{number}} محاولات تسجيل الدخول متعددة جدًا، حاول مرة أخري بعد ",
+    "Try free for {{amount}} days, then {{originalPrice}}.": "  .{{originalPrice}} يوما، ثم  {{amount}} جرب مجانًا لمدة",
+    "Unable to initiate checkout session": "غير قادر على بدء جلسة الدفع",
+    "Unlock access to all newsletters by becoming a paid subscriber.": ".افتح الوصول إلي جميع النشرات الإخبارية من خلال الاشتراك المدفوع",
+    "Unsubscribe from all emails": "إلغاء الاشتراك من جميع الالكترونية",
+    "Unsubscribed": "تم إلغاءاشتراكك",
+    "Unsubscribed from all emails.": "تم إلغاء اشتراكك من جميع الرسائل الاكترونية",
     "Unsubscribing from emails will not cancel your paid subscription to {{title}}": "الغاء اشتراك من الرسائل لا يعني إلغاء اشتراكك المدفوع في {{title}}",
-    "Update": "",
+    "Update": "تحديث",
     "Update your preferences": "تحديث تفضيلاتك",
-    "Verification link sent, check your inbox": "",
-    "Verify your email address is correct": "",
-    "View plans": "",
+    "Verification link sent, check your inbox": "تم إرسال رابط التحقق، يرجى التحقق من صندوق الوارد لديك",
+    "Verify your email address is correct": "تحقق من صحة عنوان بريدك الاكتروني",
+    "View plans": "عرض الخطط",
     "We couldn't unsubscribe you as the email address was not found. Please contact the site owner.": "لم نتمكن من إلغاء اشتراكك في الرسائل لانه لم يتم العثور على بريدك الاكتروني. برجاء التواصل مع ادارة الموقع.",
-    "Welcome back, {{name}}!": "",
-    "Welcome back!": "",
-    "Welcome to {{siteTitle}}": "",
-    "When an inbox fails to accept an email it is commonly called a bounce. In many cases, this can be temporary. However, in some cases, a bounced email can be returned as a permanent failure when an email address is invalid or non-existent.": "",
-    "Why has my email been disabled?": "",
-    "year": "",
+    "Welcome back, {{name}}!": "!{{name}} ،مرحبًا بعودتك",
+    "Welcome back!": "!مرحبًا بعودتك",
+    "Welcome to {{siteTitle}}": "{{siteTitle}} مرحبًا بك في ",
+    "When an inbox fails to accept an email it is commonly called a bounce. In many cases, this can be temporary. However, in some cases, a bounced email can be returned as a permanent failure when an email address is invalid or non-existent.": "
```

Comment:
```suggestion
    "When an inbox fails to accept an email it is commonly called a bounce. In many cases, this can be temporary. However, in some cases, a bounced email can be returned as a permanent failure when an email address is invalid or non-existent.": "عندما يفشل صندوق الوارد في استقبال رسالة بريد إلكتروني، يُطلق على ذلك عادةً ارتداد. في كثير من الحالات، قد يكون هذا الارتداد مؤقتًا. ومع ذلك، في بعض الحالات، قد يتم إرجاع البريد الإلكتروني كفشل دائم إذا كان عنوان البريد الإلكتروني غير صالح أو غير موجود."
```